tables/he-IL.utb

```diff
@@ -110,8 +110,8 @@ word 'twas =
 
 # Additional fixes for hebrew letters with dagesh
 # if the dagesh dot is typed as a separate element.
-word כּ 13 # Hebrew letter PE with dagesh as a separate dot.
-word פּ 1234 # Hebrew letter KAF with dagesh as a separate dot.
+word כּ 13 # Hebrew letter KAF with dagesh as a separate dot.
+word פּ 1234 # Hebrew letter PE with dagesh as a separate dot.
```

Comment:
this looks like it was a typo in the comment, so I thought I should fix it while I was in there.